Wangscape/Tile.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+struct Tile
```

Comment:
If you're not against extending this PR, I'll create this namespace and directory and I'll move file connected to `MetaOutput` to it.